package.json

```diff
@@ -11,55 +11,66 @@
     "format": "prettier --config .prettierrc.json --write 'src/**/*.js'",
     "format:check": "prettier --config config/.prettierrc.json --debug-check --list-different 'src/**/*.js'",
     "lint": "eslint 'src/**/*.js'",
-    "lint:fix": "eslint --fix 'src/**/*.js'"
+    "lint:fix": "eslint --fix 'src/**/*.js'",
+    "parcel": "parcel index.html",
+    "parcel-build": "parcel build index.html"
```

Comment:
these are good, but belong in the parcel PR.  could you remove them from this one? thanks!